_summaries/qiao2019micro-batch_2.md

```diff
@@ -1,19 +1,60 @@
+<p align="center"><img src="qiao2019micro-batch_2/normalization-general.png" width="1000"/></p>
+
+* Batch Normalization (`BN`) is considered to give the best results in practice whenever large batch sizes are possible but with smaller batches (which is the case when using larger models and more detailed inputs) Batch Normalization fails to perform that well because of unreliable estimates of mean and variance required for normalization.
+
+* Reason's for Batch norm's success, 1) it provides "**smooth optimization landscape**" 2) it "**avoides elimination singularity**"
+
+> Bring the success factors of Batch Norm into micro-batch training without relying on large batch-sizes during training
+
+## Weight Standardization (`WS`)
+Normalize (Standardize) weights in the convolution layer along with Group Normalization
+<p align="left"><img src="qiao2019micro-batch_2/ws-equation.png" height="200"/></p>
+
+### `WS` smoothes out the optimization landscape
+Authors theoretically argue that `WS` smoothes out the optimization by reducing the lipschitz constants on the loss and gradients
+
+Empirically they show that both the operations in `WS` (i.e. making the mean 0 and making the variance 1) help in getting better performance, with the bulk of the gains coming from the former
+
+<!-- Training ResNet-50 on Imagenet with Group Normalization (GN), Eq 11 refers to making mean 0 in WS and Eq 12 refers to making variance 1 in WS
+:----------------------------:
+<img src="qiao2019micro-batch_2/ws-ablation.png" height="250"/> | -->
+
+<p align="center"><img src="qiao2019micro-batch_2/ws-ablation.png" height="250"/></p>
+<p align="center">Training ResNet-50 on Imagenet with Group Normalization (GN), Eq 11 refers to making mean 0 in WS and Eq 12 refers to making variance 1 in WS</p>
+
+### `WS` avoides elimination singularities
+Intuitively `WS` is able to pass similarities in input channels to output channels. Empirically shown in two steps:
+  * `BN` by design enforces activation of each channels, thus `BN` should avoid singularities
+  * Adding `WS` to Group Normalization brings it closer to `BN` (i.e. zero-centered mean and unit variance for each channel)
+<br>
+
+Closer to `BN` is far from singularity | `WS` brings channel normalization closer to `BN`
+:-------------------------------------:|:---------------------------------------------:
+<img src="qiao2019micro-batch_2/bn-singularity.png" width="500"/> | <img src="qiao2019micro-batch_2/ws-bn.png" width="500"/>
+
+## Batch-Channel Normalization (`BCN`)
+Apply both batch and channel normalization together to activations
+* For Large Batches: `BCN`(X) = `CN`(`BN`(X))
+* For Micro Batches: Keep running estimates of mean and variance for `BN`
+
+> Channel based normalizations make estimates based normalizations possible, and estimate based normalization helps channel based normalizations avoid elimination singularities
+
+**TODO**: Add results
```

Comment:
Don't forget to add your results if you are still planning to

Reply:
Done.